tests/tracing/test_misc.py

```diff
@@ -82,197 +58,31 @@ def test_transaction_data(sentry_init, capture_events):
     assert span_data.items() >= {"spanfoo": "spanbar"}.items()
 
 
-def test_start_transaction(sentry_init):
-    sentry_init(traces_sample_rate=1.0)
-
-    # you can have it start a transaction for you
-    result1 = start_transaction(
-        name="/interactions/other-dogs/new-dog", op="greeting.sniff"
-    )
-    assert isinstance(result1, Transaction)
-    assert result1.name == "/interactions/other-dogs/new-dog"
-    assert result1.op == "greeting.sniff"
-
-    # or you can pass it an already-created transaction
-    preexisting_transaction = Transaction(
-        name="/interactions/other-dogs/new-dog", op="greeting.sniff"
-    )
-    result2 = start_transaction(preexisting_transaction)
-    assert result2 is preexisting_transaction
-
-
 def test_finds_transaction_on_scope(sentry_init):
     sentry_init(traces_sample_rate=1.0)
 
-    transaction = start_transaction(name="dogpark")
-
-    scope = sentry_sdk.get_current_scope()
-
-    # See note in Scope class re: getters and setters of the `transaction`
-    # property. For the moment, assigning to scope.transaction merely sets the
-    # transaction name, rather than putting the transaction on the scope, so we
-    # have to assign to _span directly.
-    scope._span = transaction
-
-    # Reading scope.property, however, does what you'd expect, and returns the
-    # transaction on the scope.
-    assert scope.transaction is not None
-    assert isinstance(scope.transaction, Transaction)
-    assert scope.transaction.name == "dogpark"
-
-    # If the transaction is also set as the span on the scope, it can be found
-    # by accessing _span, too.
-    assert scope._span is not None
-    assert isinstance(scope._span, Transaction)
-    assert scope._span.name == "dogpark"
-
-
-def test_finds_transaction_when_descendent_span_is_on_scope(
```

Comment:
Should we repurpose this one so that it checks that the `root_span` of a span is set correctly? Unless we already have that covered by the POTel tests

Reply:
added check to above test